src/ImageSharp/Primitives/Matrix5x4.cs

```diff
@@ -0,0 +1,459 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+#pragma warning disable SA1117 // Parameters should be on same line or separate lines
+using System;
+using System.Globalization;
+using System.Runtime.InteropServices;
+
+namespace SixLabors.ImageSharp.Primitives
+{
+    /// <summary>
+    /// A structure encapsulating a 5x4 matrix.
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential)]
+    public struct Matrix5x4 : IEquatable<Matrix5x4>
```

Comment:
Shouldn't we move this to SixLabors.Core?

Reply:
I don't think it has any practical application outside of ImageSharp.